src/plugin.js

```diff
@@ -1,6 +1,7 @@
 import { resolve, dirname, relative } from 'path';
 import { ResolverFactory, SyncNodeJsInputFileSystem } from 'enhanced-resolve';
 import { parse } from 'babylon';
+import template from 'lodash.template';
```

Comment:
I don't use lodash much and was a bit confused about which was the modern form, and which was the old fashioned form! So I guess we use `lodash/template` these days to take advantage of npm3 and tree shaking etc.?

Will change it ...